### DISCUSSION TITLE
Scores achieved in multiplayer without dying should appear in global ranking

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#23554] [Ideas] [answered]
For some reason when I achieve a new score that is a PB and beats someone in the global ranking for a difficulty of a beatmap in multiplayer, without dying, the score does not appear in the global ranking when I view it in Details.
When I achieve the same score in solo however, the score appears in the global ranking because I can see my previous PB there in the global ranking. So, it only shows my previous PB.

My new PB that I achieved in multiplayer still shows up under Local scores but just not in the global ones in the browser view.

Is this a temporary bug? Is it because in multiplayer you basically always have the No Fail mod on? I still think it should be counted if you didn't actually lose all of your health. Does this mean I can't get PP either?

On another unrelated note, why can I no longer view the global leaderboard under the Global tab of a beatmap when I can still see it when I rightclick and click `Details...`?

Finally, I never understood why enabling No Fail in solo gives you 50% less score but in multi it's always on and you don't get 50% less score.

**Comment by monochrome22** [comment, marked as answer]
Probably duplicate of this https://github.com/ppy/osu/issues/15888

**Reply by ghost** [reply]
Oof, that's pretty old. I would have a lot more PP. Before closing, I would like to know, will old scores eventually be submitted after that's fixed?

**Reply by peppy** [reply]
No. Also old scores will likely be deleted as mentioned on the front page (lazer is subject to score resets).
